jax/experimental/jax2tf/primitives_with_limited_support.md

```diff
@@ -58,7 +58,6 @@ conversion to Tensorflow.
 | nextafter | Missing TF support | Primitive is unimplemented in TF | bfloat16, float16 | CPU, GPU, TPU |
 | population_count | Missing TF support | Primitive is unimplemented in TF | uint32, uint64 | CPU, GPU, TPU |
 | qr | Missing TF support | Primitive is unimplemented in TF; this is a problem only in compiled mode (experimental_compile=True)) | complex128, complex64 | CPU, GPU, TPU |
-| reduce_window_sum | Missing TF support | Primitive is unimplemented in TF | uint16, uint32, uint64 | CPU, GPU, TPU |
```

Comment:
@bchetioui Do you know why this was removed? It seems to still be declared in correctness_stats.

Reply:
I figured it out. I told you for #4589 to make the initial value dynamic in the tests for reduce_window. It turns out that lax.reduce_window will only pick special reducers if the initial  value is a constant. So, it has to be StaticArg. I fixed that and new stuff appeared in the documentation.

Reply:
Oh, see my response below, we were looking into the same thing :)